modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -0,0 +1,435 @@
++
+This option is not recommended for production use cases. Many catalog services such as https://docs.databricks.com/en/data-governance/unity-catalog/index.html[Databricks Unity^] and https://github.com/apache/polaris[Apache Polaris^] provide Iceberg REST endpoints to simplify your data lakehouse management.
+
+For an Iceberg REST catalog, set the following additional cluster configuration properties:
+
+* `iceberg_rest_catalog_endpoint`: The endpoint URL for your Iceberg catalog, which you are either managing directly, or is managed by an external catalog service.
+* `iceberg_rest_catalog_client_id`: The ID to connect to the REST server.
+* `iceberg_rest_catalog_client_secret`: The secret data to connect to the REST server.
++
+--
+For REST catalogs that use self-signed certificates, also configure these properties:
+
+* `iceberg_rest_catalog_trust_file`: The path to a file containing a certificate chain to trust for the REST catalog.
+* `iceberg_rest_catalog_crl_file`: The path to the certificate revocation list for the specified trust file.
+--
+
+// update xref when PR for extracted properties is ready
+See xref:reference:properties/cluster-properties.adoc[Cluster Configuration Properties] for the full list of cluster properties to configure for a catalog integration.
+
+=== Example catalog integration
+
+You'll be able to use to the catalog to load, query, or refresh the Iceberg data as you produce to the Redpanda topic. Refer to the official documentation of your query engine or Iceberg-compatible tool for guidance on integrating your Iceberg catalog. 
+
+==== REST catalog 
+
+For example, if you have Redpanda cluster configuration properties set to connect to a REST catalog named `streaming`:
+
+[,yaml]
+----
+iceberg_rest_catalog_type: rest 
+iceberg_rest_catalog_endpoint: http://catalog-service:8181 
+iceberg_rest_catalog_client_id: <rest-connection-user>
+iceberg_rest_catalog_client_secret: <rest-connection-password>
+----
+
+And you use Spark as a processing engine, configured to use the `streaming` catalog:
+
+```
+spark.sql.catalog.streaming = org.apache.iceberg.spark.SparkCatalog
+spark.sql.catalog.streaming.type = rest
+spark.sql.catalog.streaming.uri = http://catalog-service:8181
+```
+
+Using Spark SQL, you can query the Iceberg table directly by specifying the catalog name:
+
+[,sql]
+----
+SELECT * FROM streaming.redpanda.ClickEvent;
+----
+
+Spark can use the REST catalog to automatically discover the topic's Iceberg table. 
+
+==== File system-based catalog (`object_storage`)
+
+If you are using the `object_storage` catalog type, you must set up the catalog integration in your processing engine accordingly. For example, you can configure Spark to use a file system-based catalog with at least the following properties:
+
+```
+spark.sql.catalog.streaming.type = hadoop
+spark.sql.catalog.hadoop_prod.warehouse = s3a://<bucket-name>/path/to/redpanda-iceberg-table
```

Comment:
@kbatuigas I believe this is where you need to plug the additional configs I DM-ed you.